src/components/tooltip/__tests__/tooltip.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react'
+import { ToolTip } from '../index'
+
+describe('ToolTip', () => {
```

Comment:
**nit-pick:** Tooltip is one word, not two, so I don't think Tip should be capitalised